recipe/recipe.yaml

```diff
@@ -0,0 +1,56 @@
+schema_version: 1
+
+context:
+  name: atomic-counter
+  version: 0.1.4
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/a/atomic_counter/atomic_counter-${{ version }}.tar.gz
+  sha256: c221aa80581b98c288c6833ac06e0cf74e2081bd40e0e5dcda67f590bc2739bb
+
+build:
+  number: 0
+  skip: is_abi3 and not is_python_min
+  script: ${{ PYTHON }} -m pip install . -vv
+  python:
+    version_independent: ${{ is_abi3 }}
+
+requirements:
+  build:
+    - ${{ compiler('rust') }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib("c") }}
+    - maturin
+  host:
+    - python
+    - pip
+    - maturin
+    - if: is_abi3
+      then: python-abi3
+  run:
+    - python
+
+tests:
+  - python:
+      imports:
+        - atomic_counter
```

Comment:
This test automatically does `pip check` if you don't set `pip_check: false`.

If you want to test against `python_min`, you can use `python_version: ${{ python_min }}`.

Reply:
Ah okay. This recipe was created using `conda_recipe_manager` which probably needs to know about this.

Reply:
In v1 recipes, how do you say the following
```yaml
python_version: {{ python_min }}.*  # [is_abi3]
```

Reply:
Does your solution work? `${{ python_min + ".*" if is_abi3 else "" }}`?
I see that the pipeline doesn't run the v1 recipe.

Reply:
It does.

>  I see that the pipeline doesn't run the v1 recipe.

Yeah, the rerendering is still wrong with the v1 recipe. I can't seem to figure out why

Reply:
Did you update rattler-build to the latest version? 0.35.6 has the potential to work :)

Reply:
Yep, I tried `0.35.6`

Reply:
I think part of the problem might be that `true` in variant config comes out as a string. 

As it seems like this is not what we want going forward, I'll reconsider to add `boolean` as type there. I think there is already an open PR.

This at least skips correctly:

```
  skip: is_abi3 == "true" and not is_python_min == "true"
```